### PR TITLE
Docs build warnings correction

### PR DESCRIPTION
**Pull request by alexey-khorkin**
This PR contains:
* Remove warning messages while docs building
* Remove duplication files
* Other docs fixes

Old output for `make clean html` command:
```sh
Removing everything under '_build'...
Running Sphinx v4.3.2
making output directory... done
WARNING: html_static_path entry '_static' does not exist
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 38 source files that are out of date
updating environment: [new config] 38 added, 0 changed, 0 removed
reading sources... [100%] troubleshooting                                                                                                                                                                             
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/install.rst:13: WARNING: toctree contains reference to nonexisting document '`install_package`_'
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/install.rst:13: WARNING: toctree contains reference to nonexisting document '`install_docker`_'
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/install.singularity.rst:7: WARNING: Title underline too short.

Singularity Installation
###################
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/overview.rst:8: WARNING: toctree contains reference to nonexisting document '`definitions_and_conventions`_'
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/overview.rst:8: WARNING: toctree contains reference to nonexisting document '`definitions_and_conventions`_'
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/overview.what_is_federated_learning.rst:24: WARNING: duplicate label definitions_and_conventions, other instance in /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/overview.rst
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst:8: WARNING: Duplicate explicit target name: "tensorflow".
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst:8: WARNING: Duplicate explicit target name: "tensorflow".
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst:8: WARNING: Duplicate explicit target name: "pytorch".
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst:8: WARNING: Duplicate explicit target name: "pytorch".
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst:839: WARNING: Malformed table.

+---------------------------+--------------------------------------------------+
| File Type                 | Filename                                         |
+===========================+==================================================+
| Certificate chain         | WORKSPACE.PATH/cert/cert_chain.crt               |
+---------------------------+--------------------------------------------------+
| Aggregator certificate    | WORKSPACE.PATH/cert/server/agg_{AFQDN}.crt         |
+---------------------------+--------------------------------------------------+
| Aggregator key            | WORKSPACE.PATH/cert/server/agg_{AFQDN}.key         |
+---------------------------+--------------------------------------------------+
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst:555: WARNING: duplicate label plan_settings, other instance in /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/plan_settings.rst
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst:939: WARNING: duplicate label running_the_federation:on the aggregator node, other instance in /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst:1003: WARNING: duplicate label running_the_federation:prerequisites, other instance in /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst:1026: WARNING: duplicate label running_the_federation:prerequisites, other instance in /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst:1031: WARNING: duplicate label running_the_federation:procedure, other instance in /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/openfl/components.rst:10: WARNING: toctree contains reference to nonexisting document 'source/openfl/`Short-Lived Components`_'
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/openfl/components.rst:10: WARNING: toctree contains reference to nonexisting document 'source/openfl/`Long-Lived Components`_'
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/openfl/components.rst:10: WARNING: toctree contains reference to nonexisting document 'source/openfl/`Static Diagram`_'
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/openfl/plugins.rst:8: WARNING: toctree contains reference to nonexisting document 'source/openfl/framework_adapter_'
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/openfl/plugins.rst:8: WARNING: toctree contains reference to nonexisting document 'source/openfl/serializer_plugin_'
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/openfl/plugins.rst:8: WARNING: toctree contains reference to nonexisting document 'source/openfl/device_monitor_plugin_'
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/utilities/pki.rst:8: WARNING: toctree contains reference to nonexisting document 'source/utilities/pki_overview_'
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/utilities/pki.rst:8: WARNING: toctree contains reference to nonexisting document 'source/utilities/semi_automatic_certification_'
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/utilities/pki.rst:8: WARNING: toctree contains reference to nonexisting document 'source/utilities/manual_certification_'
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/utilities/pki.rst:63: WARNING: Explicit markup ends without a blank line; unexpected unindent.
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/utilities/pki.rst:64: WARNING: Line block ends without a blank line.
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/utilities/pki.rst:74: WARNING: Explicit markup ends without a blank line; unexpected unindent.
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/utilities/pki.rst:82: WARNING: Explicit markup ends without a blank line; unexpected unindent.
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/utilities/pki.rst:95: WARNING: Explicit markup ends without a blank line; unexpected unindent.
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/utilities/pki.rst:96: WARNING: Line block ends without a blank line.
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/utilities/splitters_data.rst:52: WARNING: Content block expected for the "note" directive; none found.
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/director_based_workflow.rst:8: WARNING: duplicate label director_workflow, other instance in /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/director_based_workflow.interactive_api.rst:8: WARNING: duplicate label interactive_api, other instance in /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/director_based_workflow.interactive_api.rst:46: WARNING: duplicate label federation_api_define_fl_experiment, other instance in /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/director_based_workflow.interactive_api.rst:68: WARNING: duplicate label federation_api, other instance in /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/director_based_workflow.interactive_api.rst:101: WARNING: duplicate label experiment_api, other instance in /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/director_based_workflow.interactive_api.rst:244: WARNING: duplicate label federation_api_start_fl_experiment, other instance in /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/director_based_workflow.interactive_api.rst:301: WARNING: duplicate label federation_api_observe_fl_experiment, other instance in /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/director_based_workflow.interactive_api.rst:309: WARNING: duplicate label federation_api_complete_fl_experiment, other instance in /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/director_based_workflow.roles.rst:8: WARNING: duplicate label establishing_federation_director, other instance in /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/director_based_workflow.roles.rst:72: WARNING: duplicate label establishing_federation_envoy, other instance in /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/director_based_workflow.roles.rst:147: WARNING: duplicate label establishing_federation_experiment_manager, other instance in /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/running_the_federation.agg_based.rst:8: WARNING: duplicate label running_the_federation_aggregator_based, other instance in /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/running_the_federation.baremetal.rst:8: WARNING: Duplicate explicit target name: "pytorch".
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/running_the_federation.baremetal.rst:8: WARNING: duplicate label creating_workspaces, other instance in /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/running_the_federation.certificates.rst:19: WARNING: duplicate label install_certs_agg, other instance in /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/running_the_federation.certificates.rst:114: WARNING: duplicate label install_certs_colab, other instance in /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/running_the_federation.docker.rst:8: WARNING: duplicate label running_the_federation_docker, other instance in /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/running_the_federation.docker.rst:19: WARNING: duplicate label running_the_federation_docker_base_image, other instance in /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/running_the_federation.docker.rst:44: WARNING: duplicate label running_the_federation_docker_workspace, other instance in /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/running_the_federation.docker.rst:47: WARNING: duplicate label source/workflow/running_the_federation.docker:prerequisites, other instance in /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/running_the_federation.docker.rst
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/running_the_federation.docker.rst:52: WARNING: duplicate label source/workflow/running_the_federation.docker:procedure, other instance in /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/running_the_federation.docker.rst
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/running_the_federation.manual.rst:19: WARNING: Include file '/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/tests/github/test_hello_federation.sh' not found or reading it failed
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/running_the_federation.manual.rst:8: WARNING: duplicate label running_the_federation_manual, other instance in /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/running_the_federation.start_nodes.rst:8: WARNING: duplicate label running_the_federation.start_nodes, other instance in /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/running_the_federation.start_nodes.rst:24: WARNING: duplicate label running_collaborators, other instance in /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/running_the_federation.rst
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/running_the_federation.tutorial.rst:6: WARNING: Title overline too short.

************************************************
Open Federated Learning (|productName|) Tutorials
************************************************
looking for now-outdated files... none found
pickling environment... done
checking consistency... /home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/install.singularity.rst: WARNING: document isn't included in any toctree
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/modules.rst: WARNING: document isn't included in any toctree
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/overview.how_can_intel_protect_federated_learning.rst: WARNING: document isn't included in any toctree
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/overview.what_is_federated_learning.rst: WARNING: document isn't included in any toctree
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/overview.what_is_intel_federated_learning.rst: WARNING: document isn't included in any toctree
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/director_based_workflow.rst: WARNING: document isn't included in any toctree
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/running_the_federation.agg_based.rst: WARNING: document isn't included in any toctree
/home/akhorkin/Repositories/alexey-khorkin-repos/openfl/docs/source/workflow/running_the_federation.singularity.rst: WARNING: document isn't included in any toctree
done
preparing documents... done
writing output... [100%] troubleshooting                                                                                                                                                                              
generating indices... genindex done
writing additional pages... search done
copying images... [100%] _build/html/_images/kroki-ef179d72a64bec2b16b4fdcc171fddb26abed32a.svg                                                                                                                       
copying static files... done
copying extra files... done
dumping search index in English (code: en)... done
dumping object inventory... done
build succeeded, 67 warnings.

The HTML pages are in _build/html.
```

For now, output for `make clean html` command:
```sh
Removing everything under '_build'...
Running Sphinx v4.3.2
making output directory... done
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 22 source files that are out of date
updating environment: [new config] 22 added, 0 changed, 0 removed
reading sources... [100%] troubleshooting                                                                                                                                                                             
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] troubleshooting                                                                                                                                                                              
generating indices... genindex done
writing additional pages... search done
copying images... [100%] _build/html/_images/kroki-ef179d72a64bec2b16b4fdcc171fddb26abed32a.svg                                                                                                                       
copying static files... done
copying extra files... done
dumping search index in English (code: en)... done
dumping object inventory... done
build succeeded.

The HTML pages are in _build/html.
```

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -94,6 +94,12 @@
 # directories to ignore when looking for source files.
 # This pattern also affects html_static_path and html_extra_path.
 exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store', 'tutorials/*', 'graveyard/*']
+# add temporary unused files
```
Comment (aleksandr-mokrov) [inline docs/conf.py:74]: I suggest to remove if-else statement above (`PRODUCT_VERSION == 'Intel'`), it looks outdated

**Review by aleksandr-mokrov — APPROVED** [review]

**Comment by alexey-khorkin** [comment]
Jenkins please retry a build

**Comment by alexey-khorkin** [comment]
Jenkins please retry a build

**Comment by alexey-khorkin** [comment]
Jenkins please retry a build

**Review by dmitryagapov — APPROVED** [review]

**Review by alexey-gruzdev — APPROVED** [review]

**Pull request merged**